bundle_validator/README.md

```diff
@@ -0,0 +1,33 @@
+# Validate STIX bundle
+
+You can easily validate your stix bundle file by following the below steps:
+
+## Prerequisites:
+
+Make sure you have Python 3 installed and a shell terminal available in your machine. 
+
+If you have not already cloned the [stix-shifter github project](https://github.com/opencybersecurityalliance/stix-shifter), you need to install GIT in order to clone the STIX-shifter git repository:
```

Comment:
You should just mention that you need git installed in the line above. Right now it looks like the following terminal command is to install git, but it's really to clone the repo.

Reply:
updated